packages/database/src/migrations/20201026223059-UpdateLinkOnFeedItems-modifies-data.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function(db) {
+  /**
+   * Update value with key `link` in `template_variables`
+   */
+  return db.runSql(`
+    update feed_item
+    set template_variables = regexp_replace (
+                      template_variables :: text,
+                      '(?<=https:\\/\\/mobile.tupaia.org\\/)(country|facility)',
```

Comment:
I suppose technically we ought to be escaping the `.` character too since that's a regex special character, so it should be:
`(?<=https:\\/\\/mobile\\.tupaia\\.org\\/)(country|facility)`

But I don't think it will make any difference realistically so feel free to skip it 😄

Reply:
Thanks Rohan! It is nice to think carefully!